webrender/src/device/gfx.rs

```diff
@@ -1767,7 +1773,7 @@ impl<B: hal::Backend> Device<B> {
         let limits = adapter
             .physical_device
             .limits();
-        let max_texture_size = 4096u32; // TODO use limits after it points to the correct texture size
+        let max_texture_size = 4400u32; // TODO use limits after it points to the correct texture size
```

Comment:
hmm, this looks weird. Why 4400?

Reply:
There is a test in metal which requires this size. It's actually 2200, but the hidpi is 2.
I checked and the OpenGL backend uses 4400 sized textures as well.

Reply:
Would be nice to fix that wrench test. I don't think there is any particular reason for this large of a test, *unless* it specifically wants to exceed the maximum texture bounds.

Reply:
I don't remember which test was (maybe in boxshadow), but i definitely remember the "large" in the test name.

But there is a TODO there, because previously, the Vulkan limits was pointing to a 3D max texture size (2048, instead of the 2D max size), which was not enough for (probably) the  same test (2200 texture size).